modules/trust-cert-existing-sensor.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+
+:_module-type: PROCEDURE
+[id="trust-cert-existing-sensor_{context}"]
+= Configuring an existing Sensor to trust custom certificates
+
+.Prerequisites
+
+* Make sure that you have downloaded the Sensor bundle.
+
+.Procedure
+
+. Unzip the Sensor bundle:
++
+[source,terminal]
+----
+$ unzip -d sensor sensor-<cluster-name>.zip
```

Comment:
I was confused on these ones. They are the default names generated by the ACS when they download the file (with hyphens), so that's why I kept it as it is. Now I think I can keep the hyphen in `sensor-` and remove from `<cluseter-name>`.